.pylintrc

```diff
@@ -218,7 +219,7 @@ class-attribute-naming-style=any
 
 # Regular expression matching correct class attribute names. Overrides class-
 # attribute-naming-style
-class-attribute-rgx=[a-z_][a-z0-9_]{1,30}$
+#class-attribute-rgx=
```

Comment:
i don't understand why this commit changes the settings in contrast to the one before?